<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>საფეხბურთო რეკორდები</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #222222;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="selectionFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>331</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2b2b2b; border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="championshipLabel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ffffff; font-size: 14px; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>ჩემპიონატი</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="championshipComboBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>ინგლისის პრემიერ ლიგა</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ესპანეთის ლა ლიგა</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>იტალიის სერია ა</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>საფრანგეთის ლიგა 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ჩემპიონთა ლიგა</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="positionLabel">
       <property name="styleSheet">
        <string notr="true">color: #ffffff; font-size: 14px; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>პოზიცია</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="positionComboBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>თავდამსხმელი</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ნახევარმცველი</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>მცველი</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>მეკარე</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="recordTypeLabel_2">
       <property name="styleSheet">
        <string notr="true">color: #ffffff; font-size: 14px; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>რეკორდის ტიპი</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="recordTypeComboBox">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>გატანილი გოლები</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>საგოლე გადაცემა</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ყვითელი ბარათი</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>წითელი ბარათი</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>მშრალი კარი</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showRecordButton">
       <property name="styleSheet">
        <string notr="true">background-color: #4CAF50; color: white; border-radius: 5px; padding: 10px;
</string>
       </property>
       <property name="text">
        <string>რეკორდის ჩვენება</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="displayFrame">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>471</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> #3a3a3a; border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="recordTitleLabel">
       <property name="styleSheet">
        <string notr="true">color: #ffffff; font-size: 18px; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>საფეხბურთო რეკორდი</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="recordDisplayLabel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #cccccc; font-size: 16px; word-wrap: true;</string>
       </property>
       <property name="text">
        <string>აირჩიეთ ჩემპიონატი, პოზიცია და რეკორდის ტიპი...</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="recordDescriptionLabel">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #aaaaaa; font-size: 14px; word-wrap: true;</string>
       </property>
       <property name="text">
        <string>ინფორმაცია აქ გამოჩნდება.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: #333333; color: #ffffff;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
